Products/PloneBooking/skins/plonebooking_templates/plonebooking_add_form.cpt

```diff
@@ -15,7 +15,7 @@
     <div metal:fill-slot="main"
          tal:define="errors options/state/getErrors;
                      bcenter here/getBookingCenter;
-         	     bookable_objects python:here.getBookableObjects()">
+         	     bookable_objects python:here.getBookableObjects(review_state='published')">
```

Comment:
This seems like a fairly important change in behaviour. Can this not be done some other way?